### PR TITLE
Branch email

### LINKED ISSUE #92: Generate emails related to task contacts
As a user, I can get the emails of people related to a task so that I can easily paste it into my email application to contact them.

### PR DESCRIPTION
**Pull request by snss231**
fixes #92

Refer to the UG changes for functionality details

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Using NUS Classes can get your contact management tasks done faster than traditi
         * [Mark a task as done](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#mark-a-task-as-done-mark)
         * [Unmark a task as not done](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#unmark-a-task-as-not-done-unmark)
         * [Deleting tasks](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#deleting-tasks-deletet)
+        * [Generating emails of all contacts related to a task](https://github.com/AY2122S2-CS2103T-T12-4/tp/blob/master/docs/UserGuide.md#gen)
```
Comment (brian16600) [inline docs/UserGuide.md:32]: Sorry I realised we don't need the direct link, can you change this to simply #gen ? Then it'll point to the local version

File: docs/UserGuide.md [hunk 2]
```diff
@@ -368,7 +369,7 @@ Examples:
 
 Deletes the specified task from the task list.
 
-Format: `deletet 1`
+Format: `deletet INDEX`
```
Comment (brian16600) [inline docs/UserGuide.md:372]: Good catch thanks

File: src/main/java/seedu/address/model/task/Task.java [hunk 3]
```diff
@@ -151,6 +151,16 @@ public int getNoOfPeople() {
         return this.people.size();
     }
 
+    /**
+     * Returns a copy-paste friendly string containing all the emails related to this task.
```
Comment (brian16600) [inline src/main/java/seedu/address/model/task/Task.java:155]: Maybe can include an example on the format? Like returns a string in the format: e123456@u.nus.edu, e2345...

**Review by brian16600 — COMMENTED** [review]
Overall looks good to me

File: docs/UserGuide.md (hunk 1 above)
Comment (snss231) [reply docs/UserGuide.md]: okie donezo

File: src/main/java/seedu/address/model/task/Task.java (hunk 3 above)
Comment (snss231) [reply src/main/java/seedu/address/model/task/Task.java:198]: donezo

**Pull request merged**